.gitignore

```diff
@@ -34,6 +34,8 @@ packages/
 *.xml~
 *.t~
 
+
```

Comment:
special character? should be removed

Reply:
Not sure.  I'm doing all of my development via VS Code in the supplied Docker container.  When it mounts the container into the local path it appears that the  /.m2/repository folder showed up with the folder name having this special character.  Git tried to add all the files in it into source control. So, I excluded it in my .gitignore.  

Open to suggestions as to how to avoid this situation, but I haven't found a solution for it yet.

Reply:
FYI, here's how it appears in VS Code:  

![capture](https://user-images.githubusercontent.com/1318561/37965700-92beadd2-318b-11e8-8706-d68a3308a054.PNG)

Reply:
Wanted to update this with an issue I just opened up:  https://github.com/swagger-api/swagger-codegen/issues/7928

This appears to be an issue with Docker for Windows and a line in the `run-in-docker.sh` script.